docs/source/demos/fraud.ipynb

```diff
@@ -111,7 +111,8 @@
     "                      objective=fraud_objective,\n",
     "                      additional_objectives=['auc', 'f1', 'precision'],\n",
     "                      max_batches=1,\n",
-    "                      optimize_thresholds=True)\n",
+    "                      optimize_thresholds=True,\n",
+    "                      random_seed=0)\n",
```

Comment:
Was this necessary change for the performance diff? Isn't this the default? :o

Reply:
Mmm it wasn't necessary, I just added it to have peace of mind haha. I can remove